modules/web/package.json

```diff
@@ -57,7 +57,7 @@
     ]
   },
   "overrides": {
-    "socks": "2.7.3"
+    "socks": "2.7.4"
```

Comment:
Why change this?

Reply:
Okay, sig-api-ui needs to be the judge of that then :D

Reply:
Please revert this change